server/src/main/resources/logback.xml

```diff
@@ -16,11 +16,20 @@
   -->
 
 <configuration scan="true" scanPeriod="60 seconds" debug="false">
+    <!-- Copied from spring-boot.jar -->
+    <conversionRule conversionWord="clr" converterClass="io.seata.server.logging.logback.ColorConverter" />
+    <conversionRule conversionWord="wex" converterClass="io.seata.server.logging.logback.WhitespaceThrowableProxyConverter" />
+    <conversionRule conversionWord="wEx" converterClass="io.seata.server.logging.logback.ExtendedWhitespaceThrowableProxyConverter" />
+    <conversionRule conversionWord="pid" converterClass="io.seata.server.logging.logback.PidConverter" />
+    <property name="CONSOLE_LOG_PATTERN" value="%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(%5p) %clr(%pid){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n%wEx"/>
+    <property name="FILE_LOG_PATTERN" value="%d{yyyy-MM-dd HH:mm:ss.SSS} %5p %pid --- [%t] %-40.40logger{39} : %m%n%wEx"/>
```

Comment:
logger length original is 56, now is 39, is there a difference?

Reply:
More info added before, so 56 to 39.
Avoid the log message can't look on the small screen.